backend/src/test/java/com/huellapositiva/api/JwtControllerShould.java

```diff
@@ -160,14 +158,15 @@ void support_multiple_sessions() throws Exception {
         CredentialsVolunteerRequestDto loginDto = new CredentialsVolunteerRequestDto(DEFAULT_EMAIL, DEFAULT_PASSWORD);
         MockHttpServletResponse loginResponse = loginRequest(mvc, loginDto);
         JwtResponseDto sessionOneJwtDto = objectMapper.readValue(loginResponse.getContentAsString(), JwtResponseDto.class);
-        Thread.sleep(1100);
 
         //WHEN
         // Login with second device
         loginRequest(mvc, loginDto);
         //THEN
         // Access token from first login has been revoked due to the second login
-        assertThrows(InvalidJwtTokenException.class, () -> jwtService.getUserDetails(sessionOneJwtDto.getAccessToken()));
+        await().atMost(1, SECONDS).untilAsserted(() ->
```

Comment:
Is this really necessary?

Reply:
Yes, because the token's most precise unit to check the expiration time are _seconds_, so after a jwt revocation, the old tokens are still valid until 1 second has passed, or rather the unit of the timestamp that refers to _seconds_ has changed.
If we, for instance, set **await().atMost(300, MILLISECONDS)** it will sometimes pass the tests and sometimes it won't so, setting the maximum wait at 1 second, we ensure the test will always pass